pyanaconda/modules/storage/bootloader/base.py

```diff
@@ -795,9 +795,9 @@ def _set_storage_boot_args(self, storage):
         swap_devices = storage.fsset.swap_devices
         dracut_devices.extend(swap_devices)
 
-        # Add resume= option to enable hibernation on x86.
+        # Add resume= option to enable hibernation.
         # Choose the largest swap device for that.
-        if blivet.arch.is_x86() and swap_devices:
```

Comment:
We do support also other architectures (s390, PPC64LE, possibly others) than x86 & ARM, so maybe lets just add `blivet.arch.is_arm()` instead to the `is_x86()` so that we don't enable hibernation on platforms that might not be able tu support it ?